### DISCUSSION TITLE
Belt printer (3D print mill) integration

### DISCUSSION THREAD

**Discussion by alexianna** [supermerill/SuperSlicer#2550] [Q&A]
Hi All!

I am a very happy user of SuperSlicer and its amazing additional feature, that made it my favourite slicer so far.

Recently I moved to a belt printer for print size reasons. I am wondering whether an integration with a belt printer (e.g. Creality CR30, Ideaformer IR 3, iFactory Pro, Sainsmart INFI20) will be possible in future releases.

Sincerely,
alexianna

**Comment by supermerill** [comment]
#300

**Reply by alexianna** [reply]
Thank you very much! I searched for belt printer but could not find this result!

Very helpful! Will keep posted with print tries
